README.md

```diff
@@ -361,6 +375,6 @@ COMMAND SendEmailValicationCode(uid UserID, code STRING) RETURNS SendCodeRespons
 
 ## TODO
 
-* Make user ID autogeneratable. This enables POST.
+* ~Make user ID autogeneratable. This enables POST.~
```

Comment:
Let's talk about it!

We can merge it as is, but I'd debate that while "enabling POST" is too much of an inner goal, enabling data types as classes is a major objective for the DSL. And then, if there is a "default constructor", it can be used by the user code.